### PR TITLE
Report a plugin status

### PR DESCRIPTION
**Pull request by mraerino**
**Which problem is this pull request solving?**

Reporting additional details about edge handlers in the deploy summary.

**Describe the solution you've chosen**

Add a plugin status.

I'm aware that markdown support for plugin status is [questioned](https://github.com/netlify/netlify-react-ui/issues/4465) but i put in a comment to keep at least links.

**Describe alternatives you've considered**

n/a

**Checklist**

Please add a `x` inside each checkbox:

- [x] The status checks are successful (continuous integration). Those can be seen below.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -208,6 +211,16 @@ module.exports = {
 
     logHandlers(handlers, EDGE_HANDLERS_SRC);
     const bundle = await bundleFunctions(mainFile, utils);
-    await publishBundle(bundle, handlers, LOCAL_OUT_DIR, IS_LOCAL, NETLIFY_API_TOKEN);
+    const uploaded = await publishBundle(bundle, handlers, LOCAL_OUT_DIR, IS_LOCAL, NETLIFY_API_TOKEN);
+
+    const summaryText = uploaded
```
Comment (ehmicky) [inline src/index.js:216]: Should there be a third message for local/CLI builds?

Comment (mraerino) [reply src/index.js:216]: ah, you're right. hmm

File: src/index.js [hunk 2]
```diff
@@ -208,6 +211,16 @@ module.exports = {
...
+      ? `${handlers.length} Edge Handlers deployed.`
+      : `${handlers.length} Edge Handlers did not change.`;
+    const logsLink = `https://app.netlify.com/sites/${process.env.SITE_NAME}/edge-handlers?scope=deployid:${process.env.DEPLOY_ID}`;
```
Comment (ehmicky) [inline src/index.js:219]: Note: `SITE_NAME` is currently `undefined` in local/CLI builds. There is a bug for it at https://github.com/netlify/build/issues/1864.
I can get this fixed soon though :+1:

**Review by ehmicky — APPROVED** [review]
This is great! :tada:

**Comment by mraerino** [comment]
@ehmicky would you mind cutting a release with this?

**Pull request merged**